osquery/config/plugins/tls.cpp

```diff
@@ -46,32 +48,29 @@ CLI_FLAG(uint64,
          0,
          "Optional interval in seconds to re-read configuration");
 
+/// Option to always do enroll at startup time
+CLI_FLAG(bool,
+         config_tls_enroll_on_startup,
```

Comment:
Ok. I changed the name of the flag to enroll_always, and moved it to enroll.cpp, along with all the other enroll behavior control flags.